charts/hivemq-operator/values.yaml

```diff
@@ -199,12 +240,11 @@ hivemq:
         <cluster>
             <enabled>true</enabled>
             <transport>
-                <udp>
+                <tcp>
```

Comment:
this should also be mentioned in the upgrade guide

Reply:
i added an upgrade guide note in the ticket, is that sufficient?